testdata/addons/addons-source.yaml

```diff
@@ -7,6 +7,4 @@ spec:
   interval: 1m0s
   ref:
     branch: master
-  secretRef:
```

Comment:
removing this will break scripts/deploy.sh I think but since that is already in need of maintenance I will update it to add secretRef if required.

Reply:
cool